test/functional/preciousblock.py

```diff
@@ -47,16 +47,16 @@ def run_test(self):
         self.log.info("Ensure submitblock can in principle reorg to a competing chain")
         self.nodes[0].generate(1)
         assert_equal(self.nodes[0].getblockcount(), 1)
-        (hashY, hashZ) = self.nodes[1].generate(2)
+        (_, hashZ) = self.nodes[1].generate(2)
```

Comment:
prefer:

```
hashZ = self.nodes[1].generate(2)[-1]
```

since it conveys the meaning better: "give me the hash of the last block generated".

same for the two examples below